peer/brontide.go

```diff
@@ -138,6 +138,14 @@ type TimestampedError struct {
 	Timestamp time.Time
 }
 
+// PeerDataStore is an interface representing the struture enabling the node
+// to carry out operations to store other peer's data.
+type PeerDataStore interface {
+	StorePeerData(data []byte, peerPub *btcec.PublicKey) error
+	MarkForDelete(peerPub *btcec.PublicKey) error
+	RetrievePeerData(peerPub *btcec.PublicKey) ([]byte, error)
```

Comment:
These should all have godocs describing the operation and its relation to the other operations in the interface.

Reply:
Thanks a lot for this, this PR is still in draft. Just wanted to get approval on the approach first before submitting a proper one.